docs/spec_draft.md

```diff
@@ -1377,6 +1378,42 @@ implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.log_plus_one
+
+### Semantics
+
+Computes element-wise logarithm of `operand` tensor plus one and produces a
+`result` tensor. More formally,
+`result[i0, ..., iR-1] = log(operand[i0, ..., iR-1] + 1)`, where `log` is the
```

Comment:
For consistency reasons, let's use the same formulation as other elementwise ops, e.g. `expm1`, which don't go into the details of the formalism.